src/layout/ToolLayout.tsx

```diff
@@ -1,5 +1,10 @@
 import { FC, ReactNode } from 'react';
 
 export const ToolLayout: FC<ReactNode> = ({ children }) => {
-  return <div className="flex w-3/5 flex-col gap-4"> {children} </div>;
+  return (
+    <div className="mx-auto flex w-full flex-col gap-4 sm:mx-auto sm:w-3/5 md:mx-auto md:w-full">
+      {' '}
+      {children}{' '}
```

Comment:
Do we need those spaces here? They're not even rendered, right?
```suggestion
      {children}
```